package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@bcgov/citz-imb-sso-js-core",
+  "version": "0.0.0",
+  "description": "BCGov JavaScript SSO integration Core",
+  "author": "CITZ IMB Common Code <citz.codemvp@gov.bc.ca>",
+  "license": "Apache-2.0",
+  "main": "build/index.js",
+  "types": "build/bundle.d.ts",
+  "scripts": {
+    "build": "npm run clean:prebuild && rollup -c rollup.config.mjs && npm run build:dts && npm run clean:postbuild",
```

Comment:
hit this error when trying to run build script: 
```
src/index.ts → build...
[!] RollupError: Could not resolve entry module "src/index.ts".
    at getRollupError (/Users/tafriese/Documents/code/citz-imb-sso-js-core/node_modules/rollup/dist/shared/parseAst.js:282:41)
    at Object.error (/Users/tafriese/Documents/code/citz-imb-sso-js-core/node_modules/rollup/dist/shared/parseAst.js:278:42)
    at ModuleLoader.loadEntryModule (/Users/tafriese/Documents/code/citz-imb-sso-js-core/node_modules/rollup/dist/shared/rollup.js:20183:32)
    at async Promise.all (index 0)
```

Reply:
Yeah it wont build yet until theres src files